server.js

```diff
@@ -0,0 +1,6 @@
+
+import { app } from './app.js'
+
+app.listen({ port: 3000 }, function () {
```

Comment:
this is great, but in the spec you should listen on port from env var, or 3000 if not set.

Reply:
done